packages/perseus/src/util.ts

```diff
@@ -618,12 +624,12 @@ const Util = {
     supportsPassiveEvents,
     captureScratchpadTouchStart,
     getImageSize,
-    getImageSizeModern: GraphieUtil.getImageSizeModern,
-    getRealImageUrl: GraphieUtil.getRealImageUrl,
-    isLabeledSVG: GraphieUtil.isLabeledSVG,
-    getBaseUrl: GraphieUtil.getBaseUrl,
-    getSvgUrl: GraphieUtil.getSvgUrl,
-    getDataUrl: GraphieUtil.getDataUrl,
+    getImageSizeModern: getImageSizeModern,
+    getRealImageUrl: getRealImageUrl,
+    isLabeledSVG: isLabeledSVG,
+    getBaseUrl: getBaseUrl,
+    getSvgUrl: getSvgUrl,
+    getDataUrl: getDataUrl,
```

Comment:
Thanks.  Eventually we'll clean up this `util.ts` file so that users of it import just what they need.